### PR TITLE
feat: removed env value and extended scope support

### PR DESCRIPTION
**Pull request by EdenComp**
Also fixes GLA-174

### CODE REVIEW

File: docs/bytecode.md [hunk 1]
```diff
@@ -28,10 +28,14 @@ Integer: 8 bytes for length + integral number as bits
 
 ## Symbols
 
-- `0x21`: Builtin + 1 byte: Operator
-- `0x22`: Function Name
+- `0x21`: Operator + 1 byte
+- `0x22`: Builtin + 1 Builtin
```
Comment (Croos3r) [inline docs/bytecode.md:32]: What do you mean exactly ?

File: src/Dreamberd/Bytecode/Encode.hs [hunk 2]
```diff
@@ -33,8 +33,8 @@ transpileString :: String -> [Char]
 transpileString str = transpileInt (length str) ++ str
 
 transpileCall :: Call -> [Char]
-transpileCall (Builtin op) = toEnum 0x21 : [toEnum (0x31 + fromEnum op)]
-transpileCall (FunctionName str) = toEnum 0x22 : transpileString str
+transpileCall (Operator op) = toEnum 0x21 : [toEnum (0x31 + fromEnum op)]
```
Comment (Croos3r) [inline src/Dreamberd/Bytecode/Encode.hs:36]: Why operator ?

**Review by Croos3r — APPROVED** [review]
LGTM

File: src/Dreamberd/Bytecode/Encode.hs (hunk 2 above)
Comment (EdenComp) [reply src/Dreamberd/Bytecode/Encode.hs:36]: Builtin represents a function (atm print, error or input)
While Operator is the list of the operators we support (+, -, *, etc)

File: docs/bytecode.md (hunk 1 above)
Comment (EdenComp) [reply docs/bytecode.md:32]: Not very clear I assume! will change

**Pull request merged**